interview-questions.md

```diff
@@ -8,64 +8,86 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. Rails follows an MVC pattern. What does that mean?
 
   Your answer:
+  MVC stands for Model, View, and Controller. It breaks up our app into 3 pieces that interact with each other. Model is what our table/info looks like, Views are what we let the user see. Controller is what allows users and us to manipulate and interact with the app.
 
   Researched answer:
+  To expand on my answer, it is commonly used for developing GUIs to separate internal information like databases and ORM models with how it's shown to the end user.
 
 
 
 2. What does it mean if an application has CRUD? What does CRUD stand for?
 
   Your answer:
+  CRUD stands for Create, Read, Update, and Delete. CRUD is the four basic things you want APIs that you build to do. Create data, read data, update existing data, and destroy data.
 
-  Researched answer:
-
+  Researched answer:  
+  Expanding on my answer, a CRUD application uses forms to get data into and out of databases. It also provides developers a framework to follow in order to create full usable models.
 
 
 3. What is a migration? Why would you use one?
 
   Your answer:
+  A migration is our way to update our rails schema. In rails you never want to update the schema directly since there are a lot of behind the scenes processes that we don't see. migrations allow us to update the schema safely without breaking the app.
```

Comment:
Totally! I think it might be worth mentioning how it leaves a readable log of changes we've made to the schema.